lttoolbox/Makefile.am

```diff
@@ -28,12 +28,13 @@ lttoolboxlib = $(prefix)/lib
 
 lttoolbox_DATA = dix.dtd dix.rng dix.rnc acx.rng xsd/dix.xsd xsd/acx.xsd
 
-LDADD = liblttoolbox$(VERSION_MAJOR).la
+LDADD = liblttoolbox$(VERSION_MAJOR).la -lpthread
```

Comment:
@TinoDidriksen I got an error `libpthread.so.0: DSO missing from command line` or something when I didn't have this here – but is this the wrong solution?

Reply:
Looks like autotools has a helper https://www.gnu.org/software/autoconf-archive/ax_pthread.html to figure out the correct way to use pthreads on a given platform.

Reply:
like https://github.com/apertium/lttoolbox/pull/161/commits/3f9963c2b7212a7590b3699c60747ea6fe153958 ?